packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,37 @@
+## Guidelines
+
+### General
+
+-   **Use links**, instead, to **redirect users to another page** of the current website or app, or to another website.
+-   Avoid using too many buttons on a page. If many buttons are to be added, **less popular** or **less important** actions may be **grouped in a “more” button** (dropdown list) or **visually styled as isolated actions or links**.
```

Comment:
Suggestion:
Avoid multiplying buttons within a given page. If you need several buttons, **less popular** or **less important** actions may be **grouped in a “more” button** (dropdown list) or **visually styled as isolated actions or links**.

I'm not sure what you mean by **visually styled as isolated actions or links** though? Maybe you could try to rephrase, and I can suggest something :)